packages/google_maps_routes_api/example/lib/main.dart

```diff
@@ -0,0 +1,111 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: avoid_print
+
+import 'package:google_maps_routes_api/routes_service.dart';
+
+void main() async {
+  final ComputeRoutesResponse? routes = await _computeRoutes();
+  print('ROUTES FETCHED:');
+  print(routes?.toJson());
+  print('\n\n');
+  final List<RouteMatrixElement>? matrixes = await _computeRouteMatrix();
+  print('ROUTE MATRIX FETCHED:');
+  print(matrixes?.map((RouteMatrixElement matrix) => matrix.toJson()));
+}
+
+Future<ComputeRoutesResponse?> _computeRoutes() async {
+  const String API_KEY =
+      String.fromEnvironment('GOOGLE_API_KEY', defaultValue: 'GOOGLE_API_KEY');
+
+  final RoutesService routesService = RoutesService(apiKey: API_KEY);
+  const Waypoint origin = Waypoint(
+    location: Location(
+      latLng: LatLng(37.419734, -122.0827784),
```

Comment:
Nice place to promote Oulu by using coordinates: 65.0121, 25.4651 as starting point ;)